superset-frontend/packages/superset-ui-core/test/query/processFilters.test.ts

```diff
@@ -115,20 +113,6 @@ describe('processFilters', () => {
             operator: '==',
             comparator: 'almond',
           },
-          {
-            expressionType: 'SIMPLE',
-            clause: 'HAVING',
-            subject: 'sweetness',
-            operator: '>',
-            comparator: '0',
-          },
-          {
-            expressionType: 'SIMPLE',
-            clause: 'HAVING',
-            subject: 'sweetness',
-            operator: '<=',
-            comparator: '50',
-          },
```

Comment:
Are these Druid specific?

Reply:
I think so, because it maps to a field called `DRUID_ONLY_OPERATORS` however they seem like relatively normal operators to have.

Reply:
Ah, right, I think `expressionType=SIMPLE` is Druid-only.